sass/includes/_card-groups.scss

```diff
@@ -71,14 +69,18 @@
     }
 
     &__heading {
-      display: inline;
+      display: block;
+      margin-top: 0.3rem;
+      margin-bottom: 0.5rem;
 
       span {
         background: green;
+        display: inline;
```

Comment:
`display: inline;` won't honour padding - you'll have to use `display: inline-block;`

Reply:
💯  good spot thanks - I've updated to inline-block and also adjusted the padding ever so slightly to accommodate the line height of the text better. Such a small change it's hardly worth it :)